test/corpus/statements.txt

```diff
@@ -171,76 +282,76 @@ protected abstract def baz(type : T) : U forall T, U
 ---
 
 (source_file
-	(def
-		(method_def
-			(identifier)))
-	(def
-		(method_def
-			(identifier)
-			(constant)
-			(string
-				(quoted_string))))
-	(def
-		(method_def
-			(identifier)
-			(method_param
-				(identifier))
-			(method_param
-				(identifier))
-			(binary_operation
-				(identifier)
-				(operator)
-				(identifier))))
-	(def
-		(method_def
+  (def
```

Comment:
Not sure if my editor or the `tree-sitter test -u` did this.

If it was `tree-sitter -u`, IMO we must keep the change.

Reply:
Just checked, was `tree-sitter test -u`.